ui/apps/platform/cypress/integration/vulnerabilities/exceptionManagement/general.test.ts

```diff
@@ -0,0 +1,77 @@
+import withAuth from '../../../helpers/basicAuth';
+import { hasFeatureFlag } from '../../../helpers/features';
+import { visit } from '../../../helpers/visit';
+import { cancelAllCveExceptions } from '../workloadCves/WorkloadCves.helpers';
+import {
+    approvedDeferralsPath,
+    approvedFalsePositivesPath,
+    deferAndVisitRequestDetails,
+    deniedRequestsPath,
+    pendingRequestsPath,
+} from './ExceptionManagement.helpers';
+
+describe('Exception Management', () => {
+    withAuth();
+
+    before(function () {
+        if (
+            !hasFeatureFlag('ROX_VULN_MGMT_WORKLOAD_CVES') ||
+            !hasFeatureFlag('ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL') ||
+            !hasFeatureFlag('ROX_WORKLOAD_CVES_FIXABILITY_FILTERS')
+        ) {
+            this.skip();
+        }
+    });
+
+    beforeEach(() => {
+        if (
+            hasFeatureFlag('ROX_VULN_MGMT_WORKLOAD_CVES') &&
+            hasFeatureFlag('ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL') &&
+            hasFeatureFlag('ROX_WORKLOAD_CVES_FIXABILITY_FILTERS')
+        ) {
+            cancelAllCveExceptions();
+        }
+    });
+
+    after(() => {
+        if (
+            hasFeatureFlag('ROX_VULN_MGMT_WORKLOAD_CVES') &&
+            hasFeatureFlag('ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL') &&
+            hasFeatureFlag('ROX_WORKLOAD_CVES_FIXABILITY_FILTERS')
+        ) {
+            cancelAllCveExceptions();
+        }
+    });
+
+    it('should have the correct browser title for pending requests', () => {
+        visit(pendingRequestsPath);
+        cy.title().should('eq', 'Exception Management - Pending Requests | StackRox');
```

Comment:
Can you imitate this pattern to make assertion independent of branding?
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/cypress/integration/clusters/clusters.test.js#L23

Reply:
+1